x/name/spec/04_events.md

```diff
@@ -18,3 +19,13 @@ The name module emits the following events:
 | --------------------- | --------------------- | ------------------------- |
 | name_unbound          | name                  | {NameRecord|Name}         |
 | name_unbound          | address               | {NameRecord|Address}      |
+| name_unbound          | restricted            | {NameRecord|Restricted}   |
+
+
+### MsgBindNameRequest
```

Comment:
```suggestion
### ModifyNameProposal
```

Also, does there need to be a section in here about events from a root name proposal?

Reply:
Yes, it does emit an event. So I added it.